dataproc_jupyter_plugin/handlers.py

```diff
@@ -207,7 +207,7 @@ def gcp_service_url(self, service_name, default_url=None):
         default_url = default_url or f'https://{service_name}.googleapis.com/'
         configured_url = get_gcloud_config(f'configuration.properties.api_endpoint_overrides.{service_name}')
         url = configured_url or default_url
-        self.log.info('Service_url for service {service_name}: {url}')
+        self.log.info(f'Service_url for service {service_name}: {url}')
```

Comment:
Is the `f` an intended change

Reply:
Yes, that's a bug fix. Without the `f` this is not a format string and the template values will not be replaced with the corresponding variables' values.